ui/pages/confirmations/send/send-content/add-recipient/domain-input-resolution-cell.js

```diff
@@ -0,0 +1,200 @@
+  if (domainType === 'Other') {
+    // Snap provided resolution.
+    return (
+      <div
+        key={address}
+        className="send__select-recipient-wrapper__group-item"
+        onClick={onClick}
+      >
+        <Tooltip title={t('suggestedBy', [resolvingSnap])}>
+          <BadgeWrapper
+            badge={
+              <AvatarIcon
+                iconName={IconName.Snaps}
+                size={IconSize.Xs}
+                className="send__select-recipient-wrapper__group-item__avatar"
+                backgroundColor={IconColor.infoDefault}
+                borderColor={BackgroundColor.backgroundDefault}
+                borderWidth={2}
+                iconProps={{
+                  color: IconColor.infoInverse,
+                  style: { width: '12px', height: '12px' },
+                }}
+              />
+            }
+            positionObj={{
+              bottom: '30%',
+              right: '30%',
+              transform: 'scale(1) translate(25%, 70%)',
+              width: '20px',
+              height: '20px',
+            }}
+          >
+            <Identicon address={address} diameter={32} />
+          </BadgeWrapper>
+        </Tooltip>
+        <div className="send__select-recipient-wrapper__group-item__content">
+          <div
+            ref={titleRef}
+            className="send__select-recipient-wrapper__group-item__title"
+          >
+            {isTitleOverflowing ? (
+              <OverflowingTitle />
+            ) : (
+              <Confusable asText input={domainName} />
+            )}
+          </div>
+          <Text color={TextColor.textAlternative}>{ellipsify(address)}</Text>
+          <div className="send__select-recipient-wrapper__group-item__subtitle">
+            <Text
+              color={TextColor.textAlternative}
+              variant={TextVariant.bodySm}
+            >
+              {protocol}
+            </Text>
+          </div>
+        </div>
+      </div>
+    );
+  }
+  ///: END:ONLY_INCLUDE_IF
+  const Title = () => {
+    if (domainName && isTitleOverflowing) {
+      return <OverflowingTitle />;
+    } else if (domainName && !isTitleOverflowing) {
+      return <Confusable asText input={domainName} />;
+    }
+    return ellipsify(address);
+  };
+
+  return (
+    <div
+      key={address}
+      className="send__select-recipient-wrapper__group-item"
+      onClick={onClick}
+    >
+      <Identicon address={address} diameter={32} />
+      <div className="send__select-recipient-wrapper__group-item__content">
+        <div
+          ref={titleRef}
+          className="send__select-recipient-wrapper__group-item__title"
+        >
+          <Title />
+        </div>
+        {domainName && (
+          <div className="send__select-recipient-wrapper__group-item__subtitle">
+            <Text color={TextColor.textAlternative}>{ellipsify(address)}</Text>
+          </div>
+        )}
+        {domainType === 'ENS' && (
```

Comment:
Will this ever not be the case?

Reply:
No, because there isn't always a domain resolution, a hex address can be typed in.